Game Dev/Assets/bullet1.cs

```diff
@@ -0,0 +1,35 @@
+using System;
```

Comment:
Toate fisierele cu scripturi ar trebuie sa fie in folderu Scripts (la fel si pentru fly.cs)